package.json

```diff
@@ -7,14 +7,15 @@
   "main": "dist/server.js",
   "scripts": {
     "dev": "nodemon --ignore src/api/_auto/routes.ts --watch src -e ts --exec yarn dev-start",
-    "dev-start": "tsoa routes && ts-node src/server.ts",
+    "dev-start": "tsoa routes && tsoa swagger && yarn run build && ts-node src/server.ts ",
     "build-ts": "tsc",
     "tslint": "tslint -c tslint.json --fix -p tsconfig.json",
     "serve": "node dist/server.js",
     "build": "yarn run tslint && yarn run build-ts",
     "start": "yarn run serve",
     "tsoa:swagger": "tsoa swagger",
-    "tsoa:routes": "tsoa routes"
+    "tsoa:routes": "tsoa routes",
+    "swaggerUI": "tsoa swagger && yarn run build && yarn run serve"
```

Comment:
Zupełnie nie rozumiem sensu dodawania tej komendy. SwaggerUI jest hostowany przy każdym uruchomieniu serwera, natomiast to generowania pliku z dokumentacją jest już dedykowana komenda.

Reply:
Racja, niepotrzebne. Usuwam